### ISSUE TITLE
Preserve ordering of profile-tag relationships

### ISSUE THREAD

**Issue by razzius** [razzius/hms-weave#184]
While editing a profile, I capitalized telemedicine to Telemedicine but doing so changed the order that it appeared on the profile. We should be keeping track of the order that the users specify.

**Comment by jjchen20** [comment]
Agreed. Do we have to track the order manually or can this be coded in that
editing a tag doesn't change the order?

**Comment by razzius** [comment]
Editing a tag through the admin will not change the order. Editing the tag as a user will be a bit trickier, as users can only append tags to their tag list, and they currently cannot reorder tags.
